src/scheduled_tasks.rs

```diff
@@ -147,6 +156,45 @@ fn clear_old_activities(conn: &mut PgConnection) {
   }
 }
 
+/// overwrite posts and comments 30d after deletion
+fn overwrite_deleted_posts_and_comments(conn: &mut PgConnection) {
+  info!("Overwriting deleted posts...");
+  match diesel::update(
+    post::table
+      .filter(post::deleted.eq(true))
```

Comment:
Isn't there also the  `removed` column we could check?

Reply:
`removed` is in case of mod action. I think those should be kept for transparency.